src/appmixer/figma/core/FindComments/FindComments.js

```
@@ -0,0 +1,115 @@
+        // https://www.figma.com/developers/api#comments-get
+        const { data } = await context.httpRequest({
+            method: 'GET',
+            url: `https://api.figma.com/v1/files/${fileId}/comments`,
+            headers: {
+                'Authorization': `Bearer ${context.auth.accessToken}`
+            }
+        });
+
+        let records = data.comments || [];
+        // Apply query filter if provided
+        if (query && query.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)